ParallelGravity.h

```diff
@@ -1864,11 +1856,19 @@ class TreePiece : public CBase_TreePiece {
 	/// force that its particles see due to the other particles hosted in
```

Comment:
This comment belongs below the next declaration.  Alternatively, move the declarations back to their previous order.